LiskHQ/lisk-mobile
# Change theme according to OS' default theme - Closes #968

# What was the bug or feature?
It is described at #968 

### How did I fix it?
I added react-native-appearance library to fetch OS' default theme.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
On IOS 13:
- App should apply OS' default theme: Change OS' theme to dark, open the app. The app should be in dark mode. 
- App's theme should be a priority over OS' theme: After first test, change app's theme to light and keep OS' theme dark. Close the app and open it again, the theme should be light.
On IOS under 13 or Android:
- When opening the app for the first time, the default color should be light.
- Then, change theme to dark. Close the app and open again, the dark theme should still be applied.


# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
